lib/orchestrators/join.js

```diff
@@ -17,10 +17,11 @@ const join = (dispatch) => {
 
       // TODO better checking if task/join/junction/fork
       if (task.info) {
-        console.log('Joining to task: ' + task.info.name)
+        if (task.info.props) {
+          console.log('Joining to task: ' + task.info.props.name)
```

Comment:
is the `logging` module available to use here? we can do something like

```javascript
logger.emit('debug', `Joining to task: ${task.info.props.name}`)
```

That way we can later have stdout be filtered with CLI options, like `--debug` to enable more verbose debugging